src/APIFunctions/2DPrinting.js

```diff
@@ -62,6 +65,37 @@ export async function printPage(data) {
   return status;
 }
 
+/**
+ * Log the print request
+ * @param {Object} data         Encoded file
+ * @returns {PrintApiResponse}  Containing information for if
+ *                              the page is printing
+ */
+export async function logPrintRequest(data) {
+  let status = new ApiResponse();
+  await axios.post(LOGGING_API_URL + '/api/PrintLog/addPrintLog', data)
+    .catch(() => {
```

Comment:
If the routing doesn't throw an error what will this function return? I don't see status.responseData being set anywhere